modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/search/bar/configuration.jsp

```diff
@@ -44,7 +52,17 @@ SearchBarPortletPreferences searchBarPortletPreferences = new com.liferay.portal
 
 	<liferay-frontend:edit-form-body>
 		<liferay-frontend:fieldset-group>
-			<aui:fieldset>
+			<aui:fieldset collapsible="<%= true %>" label="display-settings">
```

Comment:
I also just noticed this should be changed to `<liferay-frontend:fieldset>`

`aui:fieldset` looks like this currently:
<img width="863" alt="Screenshot 2020-01-20 14 59 45" src="https://user-images.githubusercontent.com/4496722/72762756-a0133280-3b95-11ea-9b71-44e5a248647d.png">